src/main/java/mate/academy/model/Movie.java

```diff
@@ -0,0 +1,46 @@
+package mate.academy.model;
+
+public class Movie implements Cloneable {
+    private Long id;
+    private String title;
+
+    public Movie() {
+    }
+
+    public Movie(String title) {
+        this.title = title;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    @Override
+    public Movie clone() {
+        try {
+            return (Movie) super.clone();
+        } catch (CloneNotSupportedException e) {
+            throw new RuntimeException("Can't make clone of " + this, e);
+        }
+    }
```

Comment:
why do we need this method?

Reply:
Yep. If we will not pass cloned object the constant objects could be modified and the `id` field will be initialized